api/src/main/java/com/redhat/ipaas/api/IPaasEntity.java

```diff
@@ -15,9 +15,15 @@
  */
 package com.redhat.ipaas.api;
 
-public interface IPaasEntity {
+public abstract class IPaasEntity {
 
-	public String getId();
-	public void setId(String id);
+	abstract public String getId();
+	abstract public void setId(String id);
+	
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
I'm expressing that two IPaasEntities are equal when their IDs match. Makes it easier to work with them in Lists too and just less error prone in general as they are the same object when their IDs match..